<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageOF</class>
 <widget class="QDialog" name="MessageOF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messages Ouverture/Fermeture</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="MessageOuvertureWidget" name="tabOuverture">
      <attribute name="title">
       <string>Ouverture</string>
      </attribute>
     </widget>
     <widget class="MessageFermetureWidget" name="tabFermeture">
      <attribute name="title">
       <string>Fermeture</string>
      </attribute>
     </widget>
     <widget class="MessageOuvertureWidget" name="tabOuvertureAprem">
      <attribute name="title">
       <string>Ouverture après-midi</string>
      </attribute>
     </widget>
     <widget class="MessageFermetureWidget" name="tabFermetureAprem">
      <attribute name="title">
       <string>Fermeture après-midi</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Utiliser les mêmes messages le matin que l'après-midi ?</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QRadioButton" name="ouiRadio">
        <property name="text">
         <string>oui</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nonRadio">
        <property name="text">
         <string>non</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeSaveButton">
       <property name="text">
        <string>Enregistrer et fermer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Fermer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageFermetureWidget</class>
   <extends>QWidget</extends>
   <header>messagefermeturewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageOuvertureWidget</class>
   <extends>QWidget</extends>
   <header>messageouverturewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageOF</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>MessageOF</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
